index.html

```diff
@@ -0,0 +1,139 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>SelfDev - Making Life Easy</title>
+    <link rel="stylesheet" href="./css/style.css">
+    <link href="https://fonts.googleapis.com/icon?family=Material+Icons"
+      rel="stylesheet">
+</head>
+<body>
+    <header>
+        <div class="top-bar dark-bg">
+            <ul class="topbar-menu">
+                <li class="menu-link">
+                    <a href="" class="gray">
+                        <span class="material-icons">facebook</span>
+                    </a>
+                </li>
+                <li class="menu-link">
+                    <a href="" class="gray">
+                        <span class="md material-icons">twitter</span>
+                    </a>
```

Comment:
- [ ] Good job so far 👍 In order to follow the original design, kindly replace the `twitter` and `facebook` with their respective icons. You can learn more about icons [here](https://www.w3schools.com/icons/fontawesome_icons_intro.asp). Do this for both pages 👍 
- Your design:
![Annotation 2022-05-26 084730](https://user-images.githubusercontent.com/85113567/170446718-7d2a71bd-5a8e-4639-91cc-68e00000471b.jpg)
- Required design:
![Annotation 2022-05-26 084755](https://user-images.githubusercontent.com/85113567/170446755-27d90294-0aea-45f1-8042-e1cff87b17f4.jpg)